src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import classes from "./Checkbox.module.scss";
```

Comment:
Add `import clsx from "clsx";` (This package lets you conditionally apply styles)